packages/passkeys/passkeys/example/lib/main.dart

```diff
@@ -12,21 +12,6 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       home: HomePage(),
-      theme: ThemeData(
```

Comment:
I guess that wasn't commited on purpose, right? :)

Reply:
Yes, that wasn't on purpose. I'll revert it.

Reply:
Done